var/spack/repos/builtin/packages/m4/package.py

```diff
@@ -36,6 +36,9 @@ def configure_args(self):
         spec = self.spec
         args = ['--enable-c++']
 
+        if spec.satisfies('%cce@9:') and not spec.satisfies('platform=darwin'):
+            args.append('LDFLAGS=-rtlib=compiler-rt')
```

Comment:
The cce compiler cannot be used on darwin platforms.
```suggestion
        if spec.satisfies('%cce@9:'):
            args.append('LDFLAGS=-rtlib=compiler-rt')
```